semeio/jobs/rft/trajectory.py

```diff
@@ -68,12 +88,45 @@ def __getitem__(self, i):
     def __len__(self):
         return len(self.trajectory_points)
 
+    def to_dataframe(self):
+        """Expose the trajectory data as a Pandas DataFrame.
+
+        The data grid_ijk in self is a tuple, this is split into
+        three distinct columns, i, j and k.
+
+        Returns:
+            pd.DataFrame
+        """
+        dframe = pd.DataFrame(data=[vars(point) for point in self.trajectory_points])
+        dframe["is_active"] = [point.is_active() for point in self.trajectory_points]
+        if "grid_ijk" in dframe:
+            ijk_df = pd.DataFrame(
+                data=dframe["grid_ijk"].tolist(), index=dframe.index, columns=["i", "j", "k"]
+            )
+            dframe = pd.concat(
+                [dframe.drop("grid_ijk", axis="columns"), ijk_df], axis="columns", sort=False
+            )
+        if "measured_depth" in dframe:
+            dframe = dframe.sort_values("measured_depth")
+        elif "true_vertical_depth" in dframe:
+            dframe = dframe.sort_values("true_vertical_depth")
+        return dframe
+
```

Comment:
Your PR looks pretty good, but there is quite a bit of logic added here with no tests, could you add that please?

Reply:
Tests added